snapcraft/main.py

```diff
@@ -19,82 +19,93 @@
 snapcraft
 
 Usage:
-  snapcraft [--version | --help] [options] COMMAND [ARGS ...]
-  snapcraft [options]
+  snapcraft [options] [--no-parallel-build]
+  snapcraft [options] init
+  snapcraft [options] pull [<part> ...]
+  snapcraft [options] build [<part> ...] [--no-parallel-build]
+  snapcraft [options] stage [<part> ...]
+  snapcraft [options] strip [<part> ...]
+  snapcraft [options] clean [<part> ...] [--step <step>]
+  snapcraft [options] snap [<directory> --output <snap-file>]
```

Comment:
It is totally valid POSIX to do it this way, you can look at the last bullet here:
https://github.com/docopt/docopt#option-descriptions-format

Also, the tests pass and we have tests for -o and <directory>. If you think it would look nicer to put them in individual brackets that's fine, but look at `git pull`:
```
SYNOPSIS
    git pull [options] [<repository> [<refspec>...]]
```